### DISCUSSION TITLE
OpenOffice

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#7] [Requests] [answered]
OpenOffice website: http://www.openoffice.org/

OpenOffice privacy policy: http://www.openoffice.org/privacy.html

Logo: https://upload.wikimedia.org/wikipedia/commons/7/70/Apache_OpenOffice_logo_and_wordmark_(2014).svg

Description: "Apache OpenOffice (AOO) is an open-source office productivity software suite. It is one of the successor projects of OpenOffice.org and the designated successor of IBM Lotus Symphony. It is a close cousin of LibreOffice and NeoOffice. It contains a word processor (Writer), a spreadsheet (Calc), a presentation application (Impress), a drawing application (Draw), a formula editor (Math), and a database management application (Base).

Apache OpenOffice's default file format is the OpenDocument Format (ODF), an ISO/IEC standard. It can also read and write a wide variety of other file formats, with particular attention to those from Microsoft Office – although, unlike LibreOffice, it cannot save documents in Microsoft's post-2007 Office Open XML formats, but only import them.

Apache OpenOffice is developed for Linux, macOS and Windows, with ports to other operating systems. It is distributed under the Apache-2.0 license. The first release was version 3.4.0, on 8 May 2012. The most recent significant feature release was version 4.1, which was made available in 2014. The project has continued to release minor updates that fix bugs, update dictionaries and sometimes include feature enhancements.

Difficulties maintaining a sufficient number of contributors to keep the project viable have persisted for several years. In January 2015 the project reported a lack of active developers and code contributions. There have been continual problems providing timely fixes to security vulnerabilities since 2015. Downloads of the software peaked in 2013 with an average of just under 148,000 per day, compared to about 50,000 in 2019 and 2020.

The current major version is 4.1, first released in 2014. The most recent maintenance release was 4.1.10 in May 4, 2021."

**Comment by mayrinck** [comment, marked as answer]
Always bringing great suggestions! 
I will add this one soon, but first I will try to add LibreOffice, as it's entry came first...

Thank you for the suggestion!

**Reply by JuliaAgostino** [reply]
Okay, that's fine. You're welcome! 🙏

**Reply by JuliaAgostino** [reply]
> Always bringing great suggestions!
> I will add this one soon, but first I will try to add LibreOffice, as it's entry came first...
> 
> Thank you for the suggestion!

Has any progress been made so far?

**Reply by mayrinck** [reply]
I'm converting this one tomorrow to post through the weekend... I hope this one works.

**Reply by JuliaAgostino** [reply]
> I'm converting this one tomorrow to post through the weekend... I hope this one works.

Great news! What about LibreOffice?
